### PR TITLE
MAT-6852 display code status

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-6852](https://jira.cms.gov/browse/MAT-6852)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx [hunk 1]
```diff
@@ -69,6 +69,28 @@ export default function ResultsSection({
...
+    } else if (status == CodeStatus.INACTIVE) {
+      return (
+        <Tooltip title="This code is inactive in this code system version">
+          <IconButton>
+            <DoDisturbOutlinedIcon />
+          </IconButton>
+        </Tooltip>
+      );
+    } else {
+      return <button className="sr-only">status not available</button>;
```
Comment (RohitKandimalla) [inline src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx]: How does this look ? Do you think we need an Icon and tooltip to signify this ?

Comment (adongare) [reply src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx]: it is hidden button only read by screen readers. but now I'm thinking if regular user can't see it, should we care about screen reader?

Comment (adongare) [reply src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/ResultsSection.tsx]: I'll just remove it

**Review by RohitKandimalla — APPROVED** [review]
